app/src/main/java/org/oppia/app/home/HomeActivity.kt

```diff
@@ -6,11 +6,14 @@ import javax.inject.Inject
 
 /** The central activity for all users entering the app. */
 class HomeActivity : InjectableAppCompatActivity() {
-  @Inject lateinit var homeActivityController: HomeActivityController
+  @Inject
+  lateinit var homeActivityController: HomeActivityController
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+
     activityComponent.inject(this)
     homeActivityController.handleOnCreate()
+
```

Comment:
Remove this line. we don't need blank lines just before closing bracket

Reply:
resolved